cpp/src/cylon/table.hpp

```diff
@@ -481,6 +481,48 @@ Status WriteParquet(const std::shared_ptr<cylon::CylonContext> &ctx,
                     std::shared_ptr<cylon::Table> &table,
                     const std::string &path,
                     const io::config::ParquetOptions &options = cylon::io::config::ParquetOptions());
+/**
+   * Local_Slice the part of table to create a single table
+   * @param tables, offset, length
+   * @return new sliced table
+   */
+Status Local_Slice(const std::shared_ptr<Table> &in, int64_t offset, int64_t length,
+              std::shared_ptr<cylon::Table> &out);
+
+/**
+ * Distributed_Slice the part of table to create a single table
+ * @param table, offset and length
+ * @return new sliced table
+ */
+
+
+Status Distributed_Slice(const std::shared_ptr<Table> &in, int64_t offset, int64_t length,
+              std::shared_ptr<cylon::Table> &out, int order);
```

Comment:
order flag is added because of the slicing operation direction. Basically, Head and Tail operation is done from
top to bottom and bottom to top. order 0 means, top to bottom and 1 means bottom to top.

Reply:
Removed flag